<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AMSamplePlateItemWidget</class>
 <widget class="QWidget" name="AMSamplePlateItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>92</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sample Position</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#AMSamplePlateItemWidget {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(233, 233, 233, 255), stop:0.502513 rgba(199, 198, 198, 255), stop:1 rgba(163, 162, 162, 255));
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="samplePictureLabel">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>80</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Pic</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="sampleName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 20pt;</string>
       </property>
       <property name="text">
        <string>[Name]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="createdLabel">
       <property name="text">
        <string>created 3 days ago, 12:39pm</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="containingLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(121, 121, 121);
font-weight: bold;
font-family: &quot;Lucida Grande&quot;;</string>
         </property>
         <property name="text">
          <string>elements</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="elementsLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>elements</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="controlNames">
     <property name="styleSheet">
      <string notr="true">color: rgb(121, 121, 121);
font-weight: bold;
font-size: 10pt;
font-family: &quot;Lucida Grande&quot;;</string>
     </property>
     <property name="text">
      <string>X:
Y:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="controlValues">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 10pt;
font-family: &quot;Lucida Grande&quot;;</string>
     </property>
     <property name="text">
      <string>3.3 mm
5.7 mm</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
